### PR TITLE
AdMob Readme Enhancement

### PR DESCRIPTION
**Pull request by branflake2267**
Add some more properties to the example for using the AdMob ads.

### CODE REVIEW

File: packages/firebase_admob/README.md [hunk 1]
```diff
@@ -10,45 +10,71 @@ available yet and testing has been limited.
...
+And in the example below, the ads are given test ad unit IDs for a quick start.
 
-Banner and interstitial ads must be created with an AdMob ad unit ID, and they
-can include targeting information:
 ```
 MobileAdTargetingInfo targetingInfo = new MobileAdTargetingInfo(
-  keywords: <String>['foo', 'bar'],
-  contentUrl: 'http://foo.com/bar.html',
+  keywords: <String>['flutterio', 'beautiful apps'],
+  contentUrl: 'https://flutterio.com',
```
Comment (mravn-google) [inline packages/firebase_admob/README.md]: That should be `https://flutter.io`?

Comment (branflake2267) [reply packages/firebase_admob/README.md]: oops, I'll schedule to get back to that after work. :)

File: packages/firebase_admob/README.md [hunk 2]
```diff
@@ -10,45 +10,71 @@ available yet and testing has been limited.
...
-  keywords: <String>['foo', 'bar'],
-  contentUrl: 'http://foo.com/bar.html',
+  keywords: <String>['flutterio', 'beautiful apps'],
+  contentUrl: 'https://flutter.io',
   birthday: new DateTime.now(),
-  childDirected: true,
+  childDirected: false,
+  designedForFamilies: false,
   gender: MobileAdGender.male, // or MobileAdGender.female, MobileAdGender.unknown
+  testDevices: <String>[], // Android emulators are considered test devices
```
Comment (branflake2267) [inline packages/firebase_admob/README.md]: added <String>...

**Comment by branflake2267** [comment]
Thanks! Updated.

**Pull request merged**